pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -2084,10 +2024,6 @@ func handleDNSImpl(ctxArg interface{}, key string,
 		ctx.DNSinitialized = true
 		return
 	}
-	// if status changed to DPCStateSuccess try to send deferred objects
```

Comment:
With this removed, how is transition to `DPCStateSuccess` handled now? Shouldn't we send kick to tickers of periodic contexts?

Reply:
I assume that everything added to the deferred queue through the `SetDeferred` will be executed to a completion through the following queue kick, see the commit: https://github.com/lf-edge/eve/pull/3207/commits/8c54d14e28164c283bd8788ff3a2eef15988997c

So we do not do any explicit queue processing.

Reply:
I try to understand what happens when EVE is without network connectivity for some time and then suddenly the connectivity is restored. This was previously explicitly handled by this if statement. I'm not sure if anything needs to happen when restored connectivity is detected.

Reply:
This is a valid point. Once something enters the queue the timer will be set to a range 1-15m. So the case you've described can lead to a delay, but not to a lost wakeup. I can add an explicit kick for dns processing, just to be on the safe side.